python/pyarrow/tests/test_substrait.py

```diff
@@ -928,11 +933,12 @@ def table_provider(names, _):
 
 
 @pytest.mark.parametrize("expr", [
-    pc.equal(ds.field("x"), 7),
-    pc.equal(ds.field("x"), ds.field("y")),
-    ds.field("x") > 50
+    "pc.equal(ds.field('x'), 7)",
+    "pc.equal(ds.field('x'), ds.field('y'))",
+    "ds.field('x') > 50"
 ])
 def test_serializing_expressions(expr):
+    expr = eval(expr)
```

Comment:
```suggestion
    pc.equal(pc.field('x'), 7)",
    pc.equal(pc.field('x'), pc.field('y'))",
    pc.field('x') > 50"
])
def test_serializing_expressions(expr):
```

The `field` nowadays lives in `pyarrow.compute` (it's only still exposed through `pyarrow.dataset` as well for back compat), but so can be imported from there and is always available at the moment.

Reply:
Ah, thanks. In that case maybe I can remote dataset from this file completely. I'll try that.